schema.xsd

```diff
@@ -38,6 +38,34 @@
         </xs:annotation>
         <xs:complexType>
             <xs:sequence>
+                <xs:element name="type" type="xs:string" minOccurs="0" maxOccurs="1">
+                    <xs:annotation>
+                        <xs:documentation>
+                            The type of geocache in terms of common Opencaching types;
+                            one of the following:
+
+                            * Traditional Cache
+                            * Multi-cache
+                            * Quiz Cache
+                            * Moving Cache
+                            * Virtual Cache
+                            * Webcam Cache
+                            * Podcast Cache
+                            * Event Cache
+                            * Own Cache
+                            * Other Cache
+
+                            The types "Quiz", "Moving", "Podcast", "Own" and "Other" are
+                            not defined for groundspeak:type and will usually be mapped to
+                            groundspeak:type "Unknown Cache", which usually will be
+                            understood as "Mystery Cache" or "Quiz Cache".
+
+                            More OC types may be added. If you encounter an OC type that
+                            your application does not understand, you should treat it as
+                            "Other Cache".
```

Comment:
Perhaps this last sentence is not necessary. If I was the developer, I would probably want to simply display the type string "as it is" in the GPX file, instead of replacing it with "Other Cache".